### DISCUSSION TITLE
Splitting the todo file

### DISCUSSION THREAD

**Discussion by andrey-helldar** [Laravel-Lang/common#45] [Q&A] [answered]
We recently added translation keys for Laravel Cashier and Laravel Nova projects. Because of this, the size of the `todo.md file has grown significantly, and GitHub does not allow you to open it immediately. We need to follow the link to open it.

What do you say if I split this file into localizations?

How it will look: there will be one file `todo.md` with a general table of translation status, as now:

![image](https://user-images.githubusercontent.com/10347617/112452072-43963f00-8d67-11eb-84c5-878b3451017f.png)

and when you click on a localization, a page with it will open.

For example:

- todo/ar.md
- todo/az.md
- ...
- todo/zh_TW.md

What do you say, @Laravel-Lang/laravel-lang?

**Comment by caouecs** [comment, marked as answer]
Yes, it will be better ;)

**Reply by andrey-helldar** [reply]
I'll do it this week.

I'll fix the tests at the same time. For some reason they work on a computer, they work on PR, but not on the "master" branch.

**Reply by andrey-helldar** [reply]
Ready: https://github.com/Laravel-Lang/lang/pull/1577

**Comment by mohamedsabil83** [comment]
I agree 👍🏽

**Comment by andrey-helldar** [comment]
I see three of us agree with this decision. Okay. I'll do it this week 😊

**Comment by masterix21** [comment]
I agree

**Comment by luisprmat** [comment]
I agree to split `todo.md` by language

**Comment by andrey-helldar** [comment]
I'm having some non-project related problems. I will finalize the code in the near future.

**Comment by luisprmat** [comment]
Congratulations @andrey-helldar, you have completely rewritten this package! Excellent work!!

**Reply by andrey-helldar** [reply]
Thanks! 😊

**Comment by mohamedsabil83** [comment]
I loved how to split the todo's files. Anyone can more focus on his language phrases.

**Reply by andrey-helldar** [reply]
Thank you!

**Comment by keljtanoski** [comment]
Great work! :)

**Reply by andrey-helldar** [reply]
Thank you 😊

**Comment by masterix21** [comment]
Great work @andrey-helldar. Thank you.

**Reply by andrey-helldar** [reply]
Thank you 😊
